react-native/android/app/src/main/java/io/keybase/ossifrage/modules/FileLogger.java

```diff
@@ -0,0 +1,89 @@
+package io.keybase.ossifrage.modules;
```

Comment:
can we talk about why this is necessary? go should be able to write logs locally w/o this